build.gradle

```diff
@@ -14,13 +14,13 @@ buildscript {
         android_components: '15.0.0',
```

Comment:
Woha, that's very old. Why do we even need this? @travis79 any chance you could file a follow-up bug for getting rid of it?

Reply:
Since you asked to check for other mismatches below, I'll go ahead and get rid of it in this PR (unless you see a problem with that).

Reply:
Oops, doesn't look like we can get rid of it [because we are using it for some tests](https://github.com/mozilla/glean/blob/e65bf5e7be9404ea3a387eb85fd0130a1da15c08/glean-core/android/build.gradle#L210).  So, would you rather I update it or file a bug to go back and refactor our code to remove the dependency?

Reply:
Let's leave it be for now and file a bug to  update or refactor :)

Reply:
[Bug filed](https://bugzilla.mozilla.org/show_bug.cgi?id=1641941)